src/basics.md

```diff
@@ -411,6 +412,45 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-simple-error-handling]: #ex-simple-error-handling
+<a name="ex-simple-error-handling"></a>
+## Handle errors correctly in main
+
+[![error-chain-badge]][error-chain] [![cat-rust-patterns-badge]][cat-rust-patterns]
+
+Handles error that occur when we try to open a file that does not exist. We do this using [error-chain], a library that takes care of a lot of boilerplate code needed in order to [handle errors in Rust].
+
+We call `Io(std::io::Error)` inside `foreign_links` to convert the type `std::io::Error` into error-chain's custom version of it.
```

Comment:
> We call `Io(std::io::Error)` 

I wouldn't use `call` in this context. It is actually a definition.

> `foreign_links` to convert the type `std::io::Error` into error-chain's custom version of it

I would rephrase it "[`foreign_links`](https://brson.github.io/error-chain/error_chain/index.html#foreign-links) to allow automatic conversion from [`std::io::Error`](https://doc.rust-lang.org/std/io/struct.Error.html) into [`error_chain!`](https://brson.github.io/error-chain/error_chain/macro.error_chain.html) defined type implementing the [`Error`](https://doc.rust-lang.org/std/error/trait.Error.html) trait."

Please note the that I've added 4x links here.